ush/atmos_ensstat.sh

```diff
@@ -78,7 +78,7 @@ for outfile in ${mean_out} ${spr_out}; do
 
     ${WGRIB2} -s "${outfile}" > "${outfile}.idx"
     err=$?
-    if (( err != 0 )); then
+    if ((err != 0)); then
```

Comment:
I know we are replacing `(( ))` with `[[ ]]` because the latter provides tracing with `set -x`.
Is there a way to check for that?